src/Microsoft.AspNetCore.Mvc.Abstractions/ApiResponseType.cs

```diff
@@ -30,7 +30,7 @@ public class ApiResponseType
         /// </summary>
         /// <remarks>
         /// Will be null if the action returns no response, or if the response type is unclear. Use
-        /// <see cref="ProducesAttribute"/> or <see cref="ProducesResponseTypeAttribute"/> on an action method
+        /// <c>Microsoft.AspNetCore.Mvc.ProducesAttribute</c> or <c>Microsoft.AspNetCore.Mvc.ProducesResponseTypeAttribute</c> on an action method
```

Comment:
See [this](https://github.com/aspnet/Mvc/pull/5579#discussion_r92670742) comment